apps/meteor/tests/end-to-end/api/14-assets.js

```diff
@@ -1,34 +1,25 @@
 import { expect } from 'chai';
-import { before, describe, it } from 'mocha';
+import { before, describe, it, after } from 'mocha';
 
 import { getCredentials, api, request, credentials } from '../../data/api-data.js';
 import { imgURL } from '../../data/interactions';
+import { updatePermission } from '../../data/permissions.helper';
 
 describe('[Assets]', function () {
 	this.retries(0);
 
 	before((done) => getCredentials(done));
 
-	it('giving "manage-assets" permission to user', (done) => {
-		request
-			.post(api('permissions.update'))
-			.set(credentials)
-			.send({ permissions: [{ _id: 'manage-assets', roles: ['admin'] }] })
-			.expect('Content-Type', 'application/json')
-			.expect(200)
-			.expect((res) => {
-				expect(res.body).to.have.property('success', true);
-			})
-			.end(done);
-	});
+	before(() => updatePermission('manage-assets', ['admin']));
```

Comment:
looks like the tests don't even touch on this permission, maybe this is not required at all?

Reply:
They were updating the setting. I just removed the "manual" update to a `before/after` approach.

```
it('giving "manage-assets" permission to user', (done) => {
		request
			.post(api('permissions.update'))
			.set(credentials)
			.send({ permissions: [{ _id: 'manage-assets', roles: ['admin'] }] })
			.expect('Content-Type', 'application/json')
			.expect(200)
			.expect((res) => {
				expect(res.body).to.have.property('success', true);
			})
			.end(done);
	});
```

Reply:
true.. my bad